docs/source/app-dev/authentication.rst

```diff
@@ -0,0 +1,52 @@
+.. Copyright (c) 2019 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Authentication
+##############
+
+When developing DAML applications using SDK tools,
+your local setup will most likely not use any authentication -
+by default, any valid ledger API request will be accepted by the sandbox.
+
+To run your application against a :doc:`deployed ledger </deploy/index>`, you will need to add authentication.
+
+Introduction
```

Comment:
Many people from the Blockchain world will be used to a model where the secret a user holds is the secret required to sign a transaction. That is not the case in DAML, and we've had a few questions that hint at a lack of understanding of that. Could we expand on that a little here, make the link to "participant nodes" and how tokens relate to ledger infrastructure crypto?